be/src/common/s3_uri.h

```diff
@@ -0,0 +1,47 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include <string>
+#include <string_view>
+
+namespace starrocks {
+
+class S3URI {
```

Comment:
better to give a correct example here.

Reply:
https://github.com/StarRocks/starrocks/issues/3793